kubeflow/trainer/rhai/traininghub_test.py

```
@@ -0,0 +1,301 @@
+        args_str = crd.args[0]
+
+        # Must contain the user function source identifier and provided kwargs
+        assert "_sample_train_func" in args_str
+        assert "epochs" in args_str and "3" in args_str
+        assert "lr" in args_str and "0.1" in args_str
+
+        # PIP install header if packages requested.
+        if trainer_cfg.packages_to_install:
+            assert "pip install" in args_str
+
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)